mm2src/coins/utxo/slp.rs

```diff
@@ -1096,6 +1096,19 @@ impl MarketCoinOps for SlpToken {
         Box::new(fut.boxed().compat())
     }
 
+    fn send_raw_tx_bytes(&self, tx: &[u8]) -> Box<dyn Future<Item = String, Error = String> + Send> {
+        let coin = self.clone();
```

Comment:
SLP is a protocol working on top of BCH, which is UTXO coin. You can use `utxo_common::send_raw_tx_bytes` here or `self.platform_coin.send_raw_tx_bytes`.